README.md

```diff
@@ -1 +1,56 @@
 # MediaMonks - Frontend Coding Standards
+
+## Definition of Done Checklist
+
+#### Before Starting feature
+- [x] Read the ticket. If no ticket is present please ask the PM for one.
+- [x] Make sure the ticket is clear and actionable. If not, reassign the ticket to your project manager until the ticket is 100% clear.
+- [x] Create a feature branch (`feature/ticket-number-feature-name`)
+
+#### General Tasks
+- [x] Double check if feature is properly working on all browsers 
+- [x] Double check if feature is properly working on all resolutions
+- [x] Review all commits and check if there is room for improvement 
+- [x] Could any of the functions you wrote be reused on other components/features ?. If so, rewrite it and restart the checklist process.
+- [x] Ask yourself in which scenarios could this fail?
+- [x] Check if you are handling possible error states. Catch them.
+- [x] Merge latest develop into branch and see if there are no conflicts. If there are conflicts please ask for help if you don't know which part of the code should stay.
+- [x] Remove unnecessary comments. 
+- [x] Check the name and semantics of all functions, properties, consts etc. Do they still make sense? Could someone that doesn't know the code understand what it is doing?
+- [x] Read your code again. Do you think it can be done better or optimized? Do it. Start process again.
+- [x] Read the description of the ticket / email again. Did you really do what is asked for? Does your change solve the issue?
+- [x] Run build tasks and see if they actually works
+- [x] Do you project have code that isn't used anymore? Throw it away!
+
```

Comment:
Addition
- [x] make sure all linting is passing

Sometimes I've checkout a project which had a lot of warnings and linting errors. Although hooks should prevent this, sometimes this is the case.